### PR TITLE
Icon change successful

### PR DESCRIPTION
**Pull request by Anshnrag02**
All the fixes regarding the logo have been made. 

WhatsApp Image 2021-05-22 at 2 17 54 PM

Resizing and shape binding has been done!

Kindly merge the PR

### CODE REVIEW

**Comment by manjotsidhu** [comment]
@Nitish145 can you approve workflow run, I would like to test the build.

File: pubspec.lock [hunk 1]
```diff
@@ -28,7 +28,7 @@ packages:
       name: async
```
Comment (manjotsidhu) [inline pubspec.lock:28]: Exclude changes from this file.

File: pubspec.yaml [hunk 2]
```diff
@@ -78,6 +78,22 @@ flutter:
     - assets/images/signup/
     - assets/images/group/
     - assets/images/profile/
+    - android/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png
```
Comment (manjotsidhu) [inline pubspec.yaml:81]: Are you sure that these android/* assets are specifically required to be included ? I don't think so

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by Anshnrag02** [comment]
Yeah I synced the assets wrongly. Made the changes! sharing them

**Comment by Anshnrag02** [comment]
Please check now @manjotsidhu

**Comment by Anshnrag02** [comment]
Yeah I corrected them.

**Comment by manjotsidhu** [comment]
I will get back to soon after I tested this personally.

**Review by manjotsidhu — COMMENTED** [review]
Looks good, let me test it out.

**Comment by Anshnrag02** [comment]
sure sure

**Review by manjotsidhu — APPROVED** [review]
LGTM, Works great on my device as well.

**Pull request merged**